include/drivers/gpio.h

```diff
@@ -55,12 +55,21 @@ extern "C" {
 /* Initializes output to a high state. */
 #define GPIO_OUTPUT_INIT_HIGH   (1U << 11)
 
+/* Initializes output based on logic level */
+#define GPIO_OUTPUT_INIT_LOGICAL (1U << 12)
+
 /** @endcond */
 
 /** Configures GPIO pin as output and initializes it to a low state. */
 #define GPIO_OUTPUT_LOW         (GPIO_OUTPUT | GPIO_OUTPUT_INIT_LOW)
 /** Configures GPIO pin as output and initializes it to a high state. */
 #define GPIO_OUTPUT_HIGH        (GPIO_OUTPUT | GPIO_OUTPUT_INIT_HIGH)
+/** Configures GPIO pin as output and initializes it to a logic 0. */
+#define GPIO_OUTPUT_INACTIVE    (GPIO_OUTPUT | GPIO_OUTPUT_INIT_LOW \
+				 | GPIO_OUTPUT_INIT_LOGICAL)
```

Comment:
Minor detail: other defines in this file put `|` at the end of the line.

Reply:
I'm not alone in finding that practice maintainer-hostile, but I've made the change for consistency.